indexer/msk.tf

```diff
@@ -28,7 +28,7 @@ resource "aws_msk_cluster" "main" {
     instance_type = var.msk_instance_type
     storage_info {
       ebs_storage_info {
-        volume_size = var.environment == "mainnet" ? 2000 : 1000 # in GB
+        volume_size = var.environment == "mainnet" ? 2000 : (contains(["dev", "dev2", "dev3", "dev4", "dev5"], var.environment) ? 500 : 1000) # in GB
```

Comment:
can we pull this out into a separate variable? 

default should be 500 and then have staging/testnet use 1000 and then mainnet to use 2000.

i think using the environment variable names to determine the size is a bit hard to maintain

Reply:
Good idea. I have created a new variable in this PR https://github.com/dydxprotocol/v4-infrastructure/pull/123 and merged.